recipes/semibin/meta.yaml

```diff
@@ -21,6 +21,7 @@ requirements:
     - pip
     - python
   run:
+    - cmake 3.19.6
```

Comment:
Does this really need cmake to run?

Reply:
Thanks!. Not necessary, I think. I can remove it, if you think so.

Reply:
Yes, please do